packages/zmarkdown/.babelrc

```diff
@@ -0,0 +1,9 @@
+{
+  "presets": ["env"],
```

Comment:
Please add [`targets`](https://babeljs.io/docs/en/babel-preset-env#targets) in here

Reply:
I'd say `> 0.5%, last 2 versions, Firefox ESR, not dead, ie >=11`